Installation error on gentoo

I got an error at `pip install -U atari-py`, and the same error when trying to build from master : 
```running bdist_wheel
running build
make: Entering directory '/home/dargor/contrib/atari-py/atari_py/ale_interface'
mkdir -p build && cd build && cmake .. && make -j4
CMake Warning (dev) at CMakeLists.txt:210 (FILE):
  Policy CMP0053 is not set: Simplify variable reference and escape sequence
  evaluation.  Run "cmake --help-policy CMP0053" for policy details.  Use the
  cmake_policy command to set the policy and suppress this warning.

  For input:

    '
    IF(NOT EXISTS \"@CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt\")
      MESSAGE(FATAL_ERROR \"Cannot find install manifest: @CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt\")
    ENDIF(NOT EXISTS \"@CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt\")
    
    FILE(READ \"@CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt\" files)
    STRING(REGEX REPLACE \"\\n\" \";\" files \"\${files}\")
    FOREACH(file \${files})
      MESSAGE(STATUS \"Uninstalling \"\$ENV{DESTDIR}\${file}\"\")
      IF(EXISTS \"\$ENV{DESTDIR}\${file}\")
        EXEC_PROGRAM(
          \"@CMAKE_COMMAND@\" ARGS \"-E remove \"\$ENV{DESTDIR}\${file}\"\"
          OUTPUT_VARIABLE rm_out
          RETURN_VALUE rm_retval
          )
        IF(NOT \"\${rm_retval}\" STREQUAL 0)
          MESSAGE(FATAL_ERROR \"Problem when removing \"\$ENV{DESTDIR}\${file}\"\")
        ENDIF(NOT \"\${rm_retval}\" STREQUAL 0)
      ELSE(EXISTS \"\$ENV{DESTDIR}\${file}\")
        MESSAGE(STATUS \"File \"\$ENV{DESTDIR}\${file}\" does not exist.\")
      ENDIF(EXISTS \"\$ENV{DESTDIR}\${file}\")
    ENDFOREACH(file)
    '

  the old evaluation rules produce:

    '
    IF(NOT EXISTS "/home/dargor/contrib/atari-py/atari_py/ale_interface/build/install_manifest.txt")
      MESSAGE(FATAL_ERROR "Cannot find install manifest: /home/dargor/contrib/atari-py/atari_py/ale_interface/build/install_manifest.txt")
    ENDIF(NOT EXISTS "/home/dargor/contrib/atari-py/atari_py/ale_interface/build/install_manifest.txt")
    
    FILE(READ "/home/dargor/contrib/atari-py/atari_py/ale_interface/build/install_manifest.txt" files)
    STRING(REGEX REPLACE "\n" ";" files "${files}")
    FOREACH(file ${files})
      MESSAGE(STATUS "Uninstalling "$ENV{DESTDIR}${file}"")
      IF(EXISTS "$ENV{DESTDIR}${file}")
        EXEC_PROGRAM(
          "/usr/bin/cmake" ARGS "-E remove "$ENV{DESTDIR}${file}""
          OUTPUT_VARIABLE rm_out
          RETURN_VALUE rm_retval
          )
        IF(NOT "${rm_retval}" STREQUAL 0)
          MESSAGE(FATAL_ERROR "Problem when removing "$ENV{DESTDIR}${file}"")
        ENDIF(NOT "${rm_retval}" STREQUAL 0)
      ELSE(EXISTS "$ENV{DESTDIR}${file}")
        MESSAGE(STATUS "File "$ENV{DESTDIR}${file}" does not exist.")
      ENDIF(EXISTS "$ENV{DESTDIR}${file}")
    ENDFOREACH(file)
    '

  but the new evaluation rules produce:

    '
    IF(NOT EXISTS "@CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt")
      MESSAGE(FATAL_ERROR "Cannot find install manifest: @CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt")
    ENDIF(NOT EXISTS "@CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt")
    
    FILE(READ "@CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt" files)
    STRING(REGEX REPLACE "\n" ";" files "${files}")
    FOREACH(file ${files})
      MESSAGE(STATUS "Uninstalling "$ENV{DESTDIR}${file}"")
      IF(EXISTS "$ENV{DESTDIR}${file}")
        EXEC_PROGRAM(
          "@CMAKE_COMMAND@" ARGS "-E remove "$ENV{DESTDIR}${file}""
          OUTPUT_VARIABLE rm_out
          RETURN_VALUE rm_retval
          )
        IF(NOT "${rm_retval}" STREQUAL 0)
          MESSAGE(FATAL_ERROR "Problem when removing "$ENV{DESTDIR}${file}"")
        ENDIF(NOT "${rm_retval}" STREQUAL 0)
      ELSE(EXISTS "$ENV{DESTDIR}${file}")
        MESSAGE(STATUS "File "$ENV{DESTDIR}${file}" does not exist.")
      ENDIF(EXISTS "$ENV{DESTDIR}${file}")
    ENDFOREACH(file)
    '

  Using the old result for compatibility since the policy is not set.
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Configuring done
-- Generating done
-- Build files have been written to: /home/dargor/contrib/atari-py/atari_py/ale_interface/build
make[1]: Entering directory '/home/dargor/contrib/atari-py/atari_py/ale_interface/build'
make[1]: warning: -jN forced in submake: disabling jobserver mode.
make[2]: Entering directory '/home/dargor/contrib/atari-py/atari_py/ale_interface/build'
make[3]: Entering directory '/home/dargor/contrib/atari-py/atari_py/ale_interface/build'
make[3]: Leaving directory '/home/dargor/contrib/atari-py/atari_py/ale_interface/build'
make[3]: Entering directory '/home/dargor/contrib/atari-py/atari_py/ale_interface/build'
[  0%] Building C object CMakeFiles/ale-c-lib.dir/src/emucore/unzip.c.o
[  2%] Building CXX object CMakeFiles/ale-c-lib.dir/src/emucore/Cart3E.cxx.o
[  2%] Building CXX object CMakeFiles/ale-c-lib.dir/src/emucore/Cart2K.cxx.o
[  2%] Building CXX object CMakeFiles/ale-c-lib.dir/src/emucore/Cart3F.cxx.o
In file included from /home/dargor/contrib/atari-py/atari_py/ale_interface/src/emucore/unzip.c:13:0:
/home/dargor/contrib/atari-py/atari_py/ale_interface/src/emucore/unzip.h:114:45: error: expected '=', ',', ';', 'asm' or '__attribute__' before 'OF'
 extern int ZEXPORT unzStringFileNameCompare OF ((const char* fileName1,
                                             ^~
/home/dargor/contrib/atari-py/atari_py/ale_interface/src/emucore/unzip.h:127:32: error: expected '=', ',', ';', 'asm' or '__attribute__' before 'OF'
 extern unzFile ZEXPORT unzOpen OF((const char *path));
                                ^~
/home/dargor/contrib/atari-py/atari_py/ale_interface/src/emucore/unzip.h:138:29: error: expected '=', ',', ';', 'asm' or '__attribute__' before 'OF'
 extern int ZEXPORT unzClose OF((unzFile file));
                             ^~
/home/dargor/contrib/atari-py/atari_py/ale_interface/src/emucore/unzip.h:145:37: error: expected '=', ',', ';', 'asm' or '__attribute__' before 'OF'
 extern int ZEXPORT unzGetGlobalInfo OF((unzFile file,
                                     ^~
/home/dargor/contrib/atari-py/atari_py/ale_interface/src/emucore/unzip.h:153:40: error: expected '=', ',', ';', 'asm' or '__attribute__' before 'OF'
 extern int ZEXPORT unzGetGlobalComment OF((unzFile file,
                                        ^~
/home/dargor/contrib/atari-py/atari_py/ale_interface/src/emucore/unzip.h:166:37: error: expected '=', ',', ';', 'asm' or '__attribute__' before 'OF'
 extern int ZEXPORT unzGoToFirstFile OF((unzFile file));
                                     ^~
/home/dargor/contrib/atari-py/atari_py/ale_interface/src/emucore/unzip.h:172:36: error: expected '=', ',', ';', 'asm' or '__attribute__' before 'OF'
 extern int ZEXPORT unzGoToNextFile OF((unzFile file));
                                    ^~
/home/dargor/contrib/atari-py/atari_py/ale_interface/src/emucore/unzip.h:179:34: error: expected '=', ',', ';', 'asm' or '__attribute__' before 'OF'
 extern int ZEXPORT unzLocateFile OF((unzFile file,
                                  ^~
/home/dargor/contrib/atari-py/atari_py/ale_interface/src/emucore/unzip.h:192:42: error: expected '=', ',', ';', 'asm' or '__attribute__' before 'OF'
 extern int ZEXPORT unzGetCurrentFileInfo OF((unzFile file,
                                          ^~
/home/dargor/contrib/atari-py/atari_py/ale_interface/src/emucore/unzip.h:218:39: error: expected '=', ',', ';', 'asm' or '__attribute__' before 'OF'
 extern int ZEXPORT unzOpenCurrentFile OF((unzFile file));
                                       ^~
/home/dargor/contrib/atari-py/atari_py/ale_interface/src/emucore/unzip.h:224:40: error: expected '=', ',', ';', 'asm' or '__attribute__' before 'OF'
 extern int ZEXPORT unzCloseCurrentFile OF((unzFile file));
                                        ^~
/home/dargor/contrib/atari-py/atari_py/ale_interface/src/emucore/unzip.h:231:39: error: expected '=', ',', ';', 'asm' or '__attribute__' before 'OF'
 extern int ZEXPORT unzReadCurrentFile OF((unzFile file,
                                       ^~
/home/dargor/contrib/atari-py/atari_py/ale_interface/src/emucore/unzip.h:245:32: error: expected '=', ',', ';', 'asm' or '__attribute__' before 'OF'
 extern z_off_t ZEXPORT unztell OF((unzFile file));
                                ^~
/home/dargor/contrib/atari-py/atari_py/ale_interface/src/emucore/unzip.h:250:27: error: expected '=', ',', ';', 'asm' or '__attribute__' before 'OF'
 extern int ZEXPORT unzeof OF((unzFile file));
                           ^~
/home/dargor/contrib/atari-py/atari_py/ale_interface/src/emucore/unzip.h:255:42: error: expected '=', ',', ';', 'asm' or '__attribute__' before 'OF'
 extern int ZEXPORT unzGetLocalExtrafield OF((unzFile file,
                                          ^~
/home/dargor/contrib/atari-py/atari_py/ale_interface/src/emucore/unzip.c: In function 'unzOpen':
/home/dargor/contrib/atari-py/atari_py/ale_interface/src/emucore/unzip.c:419:5: warning: implicit declaration of function 'unzGoToFirstFile' [-Wimplicit-function-declaration]
     unzGoToFirstFile((unzFile)s);
     ^~~~~~~~~~~~~~~~
/home/dargor/contrib/atari-py/atari_py/ale_interface/src/emucore/unzip.c: In function 'unzClose':
/home/dargor/contrib/atari-py/atari_py/ale_interface/src/emucore/unzip.c:437:9: warning: implicit declaration of function 'unzCloseCurrentFile' [-Wimplicit-function-declaration]
         unzCloseCurrentFile(file);
         ^~~~~~~~~~~~~~~~~~~
/home/dargor/contrib/atari-py/atari_py/ale_interface/src/emucore/unzip.c: At top level:
/home/dargor/contrib/atari-py/atari_py/ale_interface/src/emucore/unzip.c:479:47: error: expected '=', ',', ';', 'asm' or '__attribute__' before 'OF'
 local int unzlocal_GetCurrentFileInfoInternal OF((unzFile file,
                                               ^~
make[3]: *** [CMakeFiles/ale-c-lib.dir/build.make:375: CMakeFiles/ale-c-lib.dir/src/emucore/unzip.c.o] Error 1
make[3]: *** Waiting for unfinished jobs....
make[3]: Leaving directory '/home/dargor/contrib/atari-py/atari_py/ale_interface/build'
make[2]: *** [CMakeFiles/Makefile2:100: CMakeFiles/ale-c-lib.dir/all] Error 2
make[2]: Leaving directory '/home/dargor/contrib/atari-py/atari_py/ale_interface/build'
make[1]: *** [Makefile:130: all] Error 2
make[1]: Leaving directory '/home/dargor/contrib/atari-py/atari_py/ale_interface/build'
make: *** [Makefile:4: build] Error 2
make: Leaving directory '/home/dargor/contrib/atari-py/atari_py/ale_interface'
Could not build atari-py: Command '['make', 'build', '-C', 'atari_py/ale_interface', '-j', '3']' returned non-zero exit status 2.. (HINT: are you sure cmake is installed? You might also be missing a library. Atari-py requires: zlib [installable as 'apt-get install zlib1g-dev' on Ubuntu].)
Traceback (most recent call last):
  File "setup.py", line 36, in <module>
    tests_require=['nose2']
  File "/home/dargor/python36/lib/python3.6/site-packages/setuptools/__init__.py", line 129, in setup
    return distutils.core.setup(**attrs)
  File "/home/dargor/python36/lib/python3.6/distutils/core.py", line 148, in setup
    dist.run_commands()
  File "/home/dargor/python36/lib/python3.6/distutils/dist.py", line 955, in run_commands
    self.run_command(cmd)
  File "/home/dargor/python36/lib/python3.6/distutils/dist.py", line 974, in run_command
    cmd_obj.run()
  File "/home/dargor/python36/lib/python3.6/site-packages/wheel/bdist_wheel.py", line 202, in run
    self.run_command('build')
  File "/home/dargor/python36/lib/python3.6/distutils/cmd.py", line 313, in run_command
    self.distribution.run_command(command)
  File "/home/dargor/python36/lib/python3.6/distutils/dist.py", line 974, in run_command
    cmd_obj.run()
  File "setup.py", line 16, in run
    subprocess.check_call(cmd)
  File "/home/dargor/python36/lib/python3.6/subprocess.py", line 291, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['make', 'build', '-C', 'atari_py/ale_interface', '-j', '3']' returned non-zero exit status 2.
```

Any clue about what is going on here ?

Software versions : 
`gcc (Gentoo 6.4.0-r1 p1.3) 6.4.0`
`cmake version 3.9.6`
`Python 3.6.5`

--- a/atari_py/ale_interface/src/emucore/unzip.h
+++ b/atari_py/ale_interface/src/emucore/unzip.h
@@ -41,6 +41,13 @@
 #ifndef _unz_H
 #define _unz_H
 
+/* Gentoo removed OF from their copy of zconf.h
+ * (https://bugs.gentoo.org/show_bug.cgi?id=383179) */
+/* but our copy of minizip needs it. */
+#ifndef OF
+#define OF(args) args
+#endif
+
 #ifdef __cplusplus
 extern "C" {
 #endif

